static/src/javascripts/projects/common/modules/experiments/tests/membership-message-uk.js

```diff
@@ -18,43 +18,49 @@ define([
     svgs
 ) {
 
+    var messageId = 'membership-message-uk';
+
     return function () {
 
-        this.id = 'MembershipMessageVariants';
-        this.start = '2015-07-13';
-        this.expiry = '2015-08-20';
+        this.id = 'MembershipMessageUk';
+        this.start = '2015-08-16';
+        this.expiry = '2015-09-21';
         this.author = 'David Rapson';
-        this.description = 'Test if loyal users are encouraged to join Membership as a Supporter';
+        this.description = 'Test if loyal visitors in the UK edition are encouraged to join Membership as a Supporter';
         this.audience = 1;
         this.audienceOffset = 0;
-        this.successMeasure = 'Loyal users will be interested in becoming a Supporter';
-        this.audienceCriteria = 'Users who have seen 10 or more pages, message shown only on article pages';
-        this.dataLinkNames = 'supporter message, hide, read more';
+        this.successMeasure = 'Loyal visitors will be interested in becoming a Supporter';
+        this.audienceCriteria = 'UK edition visitors who have seen at least 10 pages, message shown only on article pages';
+        this.dataLinkNames = 'membership message, hide, read more';
         this.idealOutcome = 'Users will sign up as a Supporter';
 
         this.canRun = function () {
             /**
-             * Exclude adblock users to avoid conflicts with similar adblock Supporter message,
-             * only show to users who have viewed 10 or more pages.
+             * - Exclude adblock users to avoid conflicts with similar adblock Supporter message
+             * - Exclude mobile/small-screen devices
+             * - Only show for UK edition
+             * - Only show on Article pages
+             * - Only show to visitors who have viewed at least 10 pages.
              */
             var alreadyVisited = storage.local.get('alreadyVisited') || 0;
             return !detect.adblockInUse &&
+                detect.getBreakpoint() !== 'mobile' &&
                 config.page.edition === 'UK' &&
                 config.page.contentType === 'Article' &&
-                alreadyVisited > 9;
+                alreadyVisited > 10;
```

Comment:
No it starts at 1. In this case I'm deliberately increasing the threshold so it's consistent with the US version.